File: Jack.Model/DTOs/Sacolas.vb
<Serializable()> _
Public Class Sacolas

    Public Overridable Property NumeroSacola As Integer
    Public Overridable Property NumeroSacolaFamilia
    Public Overridable Property CodigoFamilia As Integer
    Public Overridable Property CodigoFamiliaRep As Integer
    Public Overridable Property CodigoCrianca As Integer
    Public Overridable Property Sexo As String
    Public Overridable Property NomeMae As String
    Public Overridable Property NomeMaeRep As String
    Public Overridable Property NomeCrianca
    Public Overridable Property Calcado As Integer
    Public Overridable Property Roupa As String
    Public Overridable Property Idade As Integer
    Public Overridable Property MedidaIdade As String
    Public Overridable Property CodigoKit As Integer
    Public Overridable Property DescricaoKit As String
    Public Overridable Property NumeroNivel As Integer
    Public Overridable Property Impresso As String
    Public Overridable Property CodigoStatus As Integer
    Public Overridable Property DescricaoStatus As String

    Public Overridable ReadOnly Property IdadeCrianca As String
        Get

            Return Idade.ToString() + " " + MedidaIdade

        End Get
    End Property

End Class

File: Jack.Model/DTOs/Sacolas.cs
using System;

namespace Jack.Model
{
    [Serializable()]
    public class Sacolas
    {

        public virtual int NumeroSacola { get; set; }
        public virtual object NumeroSacolaFamilia { get; set; }
        public virtual int CodigoFamilia { get; set; }
        public virtual int CodigoFamiliaRep { get; set; }
        public virtual int CodigoCrianca { get; set; }
        public virtual string Sexo { get; set; }
        public virtual string NomeMae { get; set; }
        public virtual string NomeMaeRep { get; set; }
        public virtual object NomeCrianca { get; set; }
        public virtual int Calcado { get; set; }
        public virtual string Roupa { get; set; }
        public virtual int Idade { get; set; }
        public virtual string MedidaIdade { get; set; }
        public virtual int CodigoKit { get; set; }
        public virtual string DescricaoKit { get; set; }
        public virtual int NumeroNivel { get; set; }
        public virtual string Impresso { get; set; }
        public virtual int CodigoStatus { get; set; }
        public virtual string DescricaoStatus { get; set; }

        public virtual string IdadeCrianca
        {
            get { return Idade.ToString() + " " + MedidaIdade; }
        }

    }

}
